### PR TITLE
Create a new API endpoint

### PR DESCRIPTION
**Pull request by samiterefe**

### CODE REVIEW

File: plugins/webservices/rsfilesreports/src/Extension/RSFilesReports.php
Comment (kevincam3) [inline plugins/webservices/rsfilesreports/src/Extension/RSFilesReports.php:1]: You should only be adding end points but you removed an existing ones

Comment (samiterefe) [reply plugins/webservices/rsfilesreports/src/Extension/RSFilesReports.php:1]: what end points did i removed?

File: plugins/webservices/rsfilesreports/src/Extension/RSFilesReports.php [hunk 1]
```diff
@@ -10,27 +9,18 @@
...
-	 * @return  void
-	 *
-	 * @since   4.0.0
-	 */
-	public function onBeforeApiRoute(&$router): void
-	{
-		$defaults = ['public' => true, 'component' => 'com_rsfilesreports'];
-		$routes   = [
-			new Route(['GET'], 'v1/rsfilesreports/get/documents/info', 'downloaded.getDocumentsInfo', [], $defaults),
-			new Route(['POST'], 'v1/rsfilesreports/save/document/downloaded', 'downloaded.saveDownloaded', [], $defaults),
```
Comment (kevincam3) [inline plugins/webservices/rsfilesreports/src/Extension/RSFilesReports.php:29]: saveDownloaded is missing

Comment (samiterefe) [reply plugins/webservices/rsfilesreports/src/Extension/RSFilesReports.php:29]: Ok i will insert it.

**Pull request closed without merging**